userstatus/userstatuswwu/classes/userstatuswwu.php

```diff
@@ -209,7 +209,7 @@ private function get_all_ziv_users() {
                         if ($group === $membergroup) {
                             $currentname = $currentstring[0];
```

Comment:
I would like to see $currentname and $group (->rename this war to $currentgroup) to be defined outside of these foreach loop and the switch statement above. Please define it directly under the validity checks for $currentstring.